example_recursive_test.go

```diff
@@ -24,14 +26,18 @@ func recursiveServeHTTP(l net.Listener) {
 	s := &http.Server{
 		Handler: &recursiveHTTPHandler{},
 	}
-	s.Serve(l)
+	go func() {
+		if err := s.Serve(l); err != cmux.ErrListenerClosed {
+			log.Panic(err)
+		}
+	}()
 }
 
 func tlsListener(l net.Listener) net.Listener {
 	// Load certificates.
 	certificate, err := tls.LoadX509KeyPair("cert.pem", "key.pem")
 	if err != nil {
-		log.Fatal(err)
+		log.Panic(err)
```

Comment:
How does `log.Fatal(err)` compare to/differer from `panic(err)`?

From https://godoc.org/log#Panic:

> Panic is equivalent to Print() followed by a call to panic().

If I'm understanding that right, in this case, they are equivalent, since the only parameter is a single `error` value. Is that right?